src/Moq/Protected/IProtectedAsMock.cs

```diff
@@ -96,6 +131,17 @@ public interface IProtectedAsMock<T, TAnalog> : IFluentInterface
 		/// <exception cref="MockException">The specified invocation did not occur (or did not occur the specified number of times).</exception>
 		void Verify<TResult>(Expression<Func<TAnalog, TResult>> expression, Times? times = null, string failMessage = null);
 
+		/// <summary>
+		///   Verifies that a property was set on the mock, specifying a failure message.
+		/// </summary>
+		/// <param name="times">The number of times a method is expected to be called. Defaults to Times.AtLeastOnce</param>
```

Comment:
I'd suggest either removing the _"Defaults to Times.AtLeastOnce"_ (since IIRC this isn't usually mentioned anywhere else):

```suggestion
		/// <param name="times">The number of times a method is expected to be called.</param>
```

... or else cross-referencing it using `<see cref="..."/>`:

```suggestion
		/// <param name="times">The number of times a method is expected to be called. Defaults to <see cref="Times.AtLeastOnce()"/>.</param>
```